components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2267,24 +2267,42 @@ def _annotations(request, objtype, objid, conn=None, **kwargs):
     query += """
```

Comment:
@emilroz I went to add a test for loading of table data for plate. Although I didn't test the permissions in the test itself, I found that for ```root``` user to query data from another user in a private group *does* need to fetch ```links.details.owner```. Thanks!